osu.Game.Tests/Visual/UserInterface/TestSceneOsuHoverContainer.cs

```diff
@@ -20,18 +20,9 @@ public class TestSceneOsuHoverContainer : ManualInputManagerTestScene
         private OsuHoverTestContainer hoverContainer;
         private OsuSpriteText textContainer;
         private ColourInfo currentColour => textContainer.DrawColourInfo.Colour;
-        private ColourInfo idleColour => hoverContainer.IdleColourPublic;
-        private ColourInfo hoverColour => hoverContainer.HoverColourPublic;
-
-        public TestSceneOsuHoverContainer()
```

Comment:
This constructor was needed because otherwise when running test from the test GUI, the whole test suite would crash. Have you tried it before merging?

Reply:
yes